chromium_src/chrome/browser/about_flags.cc

```diff
@@ -188,8 +188,11 @@ using ntp_background_images::features::kBraveNTPSuperReferralWallpaper;
      flag_descriptions::kBraveRewardsVerboseLoggingName,                    \
      flag_descriptions::kBraveRewardsVerboseLoggingDescription,             \
      kOsDesktop | kOsAndroid,                                               \
-     FEATURE_VALUE_TYPE(brave_rewards::features::kVerboseLoggingFeature)},
-
+     FEATURE_VALUE_TYPE(brave_rewards::features::kVerboseLoggingFeature)},  \
```

Comment:
this also needs a buildflag

Reply:
added: https://github.com/brave/brave-core/pull/8566/files#diff-6b46cb9060fbbe84429044c54272db22810e8d2106bec1197f5631be28d55d61R84